openapi_python_client/parser/properties/schemas.py

```diff
@@ -14,10 +18,66 @@
     ModelProperty = "ModelProperty"
 
 
+_ReferencePath = NewType("_ReferencePath", str)
+_ClassName = NewType("_ClassName", str)
+
+
+def parse_reference_path(ref_path_raw: str) -> Union[_ReferencePath, ParseError]:
+    parsed = urlparse(ref_path_raw)
+    if parsed.scheme or parsed.path:
+        return ParseError(detail=f"Remote references such as {ref_path_raw} are not supported yet.")
+    return cast(_ReferencePath, parsed.fragment)
+
+
+@attr.s(auto_attribs=True, frozen=True)
+class Class:
+    """ Info about a generated class which will be in models """
+
+    name: _ClassName
+    module_name: str
+
+    @staticmethod
+    def from_string(*, string: str, config: Config) -> "Class":
```

Comment:
Would `from_ref` be clearer here?

Reply:
That's what the method on the old `Reference` class said, but technically this doesn't _have_ to be a ref, it can be any string, so I thought this name better indicated what the method is capable of.